deploy/mysql_db_schema.json

```diff
@@ -204,6 +204,17 @@
         {
           "dataType": "VARCHAR",
           "decimalDigits": 0,
+          "width": "(200)",
+          "name": "azure_pem_url",
+          "nullable": true,
+          "remarks": "",
+          "autoIncremented": false,
+          "databaseSpecificType": "VARCHAR",
+          "size": 200
+        },
+        {
+          "dataType": "VARCHAR",
+          "decimalDigits": 0,
```

Comment:
both `deploy/mysql_db_schema.json` and `deploy/mysql_db_schema.html` are auto-generated by tools. You don't have to update it mannully